src/assets/scss/components/download-button.scss

```
@@ -0,0 +1,33 @@
+  }
+}
+
+.app-c-download-button__button--disabled {
+  pointer-events: none;
+  opacity: 0.8;
+}
+
+.app-c-spinner {
+  width: 1rem;
+  height: 1rem;
+  border: 2px solid currentColor;
```

Stylelint (17.6.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)